MetaMorpheus/EngineLayer/GlycoSearch/Glycan.cs

```diff
@@ -659,7 +668,7 @@ public static bool Equals(Glycan glycan1, Glycan glycan2)
                     GlycanIon glycanIon = new GlycanIon(null, ion.IonMass + value, ion.IonKind, ion.LossIonMass - value);
                     glycanIons.Add(glycanIon);
                 }
-                var aDecoyGlycan = new Glycan(aGlycan.Struc, aGlycan.Mass, aGlycan.Kind, glycanIons, true);
+                var aDecoyGlycan = new Glycan(aGlycan.Struc, aGlycan.Mass, aGlycan.Kind, glycanIons, true, aGlycan.Motif, aGlycan.Type);
```

Comment:
i don't like the use of a at the start because it confuses me with the use of n and o at the start. suggest you call it decoyGlycan